packages/server-wallet/e2e-test/e2e-utils.ts

```diff
@@ -56,4 +59,21 @@ export const waitForServerToStartAndResetDatabase = (
     }, pingInterval);
   });
 
+/**
+ * Asks the Pong Server to identify itself so that Ping may use this info in the creation
+ * of a new channel with Pong.
+ */
+export const getPongsParticipantInfo = async (pongServer: PongServer): Promise<Participant> => {
+  const {data: participant} = await axios.get<Participant>(`${pongServer.url}/participant`);
+  return participant;
+};
+
+/**
+ * Seeds Pong's database with a channel.
+ */
+export const seedPongWithChannel = async (
```

Comment:
I'm not sure if the indirection of adding and hitting HTTP endpoints to seed a channel is that valuable.

One alternative is to explicitly seed the database: You can have two `knex` objects: `knexPing`, connected to the `ping` database and `knexPong`, connected to the `pong` database. You can then do
```
import {seedChannels} from './path/to/seeds/seed_channels'
import {channel} from './path/to/fixtures/channels

await seedChannels(knexPong, myChannels)
```

Reply:
Yeah I think this is better too, I'll do this.